Cargo.lock

```diff
@@ -9885,13 +9886,17 @@ dependencies = [
  "hex",
  "hex-literal",
  "log",
+ "orml-traits",
+ "orml-xcm",
+ "orml-xcm-support",
  "pallet-aura",
```

Comment:
Nice, this lock file confirms that we do not pull in the duplicate substrate deps from the dev-dependencies.

Reply:
Ah wait, this is not relevant yet, we have to hope that we don't see the deps in the other PR. :)